### PR TITLE
[HW1] mylin0405 hw1

### PR DESCRIPTION
**Pull request by mylin0405**
---
A good PR should include the following parts:

- [ ] A clear title
- [ ] A meaningful message for PR, as well as its commits
- [ ] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [ ] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

File: hw0/README.rst [hunk 1]
```diff
@@ -44,6 +44,6 @@ run the following commands:
 .. code-block:: bash
 
   cd nsdhw_21au/hw0/username
-  ../validate.py | grep "GET POINT"
+  ../validate.sh | grep "GET POINT"
```
Comment (yungyuc) [inline hw0/README.rst:47]: Please do not include changes of other branch.  It seems that this PR did not branch from `hw1`.

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: hw0/README.rst (hunk 1 above)
Comment (mylin0405) [reply hw0/README.rst]: Thank you. I have changed the wrong commit.

Comment (yungyuc) [reply hw0/README.rst]: No you didn't.  See https://github.com/yungyuc/nsdhw_21au/pull/20/commits .  The unwanted change is still in your branch.

Comment (mylin0405) [reply hw0/README.rst]: Thanks. I'll close this PR and re-branch it.

**Pull request closed without merging**